core/src/identity/database/user-authentication-account.repository.ts

```diff
@@ -13,6 +13,7 @@ export class UserAuthenticationAccountRepository extends ExtendedRepository<User
         super(UserAuthenticationAccount, dataSource);
     }
 
+    // TODO: Allow multiple Discord accounts? How do we handle?
```

Comment:
Do we have a ticket to at least discuss this? Might as well

Reply:
Ticketed